packages/spacecat-shared-data-access/src/service/import-url/accessPatterns.js

```diff
@@ -117,3 +117,37 @@ export const getImportUrlsByJobId = async (dynamoClient, config, log, jobId) =>
 
   return items ? items.map((item) => ImportUrlDto.fromDynamoItem(item)) : [];
 };
+
+/**
+ * Remove all given import URLs.
+ * @param {DynamoDbClient} dynamoClient - The DynamoDB client.
+ * @param {DataAccessConfig} config - The data access config.
+ * @param {ImportUrl[]} urls - The import URLs to remove.
+ * @return {Promise<void>} A promise that resolves when all URLs have been removed.
+ */
+async function removeUrls(dynamoClient, config, urls) {
+  const removeUrlPromises = urls.map((url) => dynamoClient.removeItem(
```

Comment:
If we are going to use it just for the tests, it might not be necessary. But, something to consider: Do we want to use batch delete if we are removing multiple URLs at the same time? This will need an update in the dynamo client: https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/dynamodb/command/BatchWriteItemCommand/

Reply:
Initially we will only be adding this API key scope to the test API keys. Good to know that a batch delete exists if we want to support this on a larger, non-test related scale in the future.